src/renderer/components/+extensions/__tests__/extensions.test.tsx

```diff
@@ -20,36 +20,27 @@ import { DiRender, renderFor } from "../../test-utils/renderFor";
 import extensionDiscoveryInjectable from "../../../../extensions/extension-discovery/extension-discovery.injectable";
 import directoryForUserDataInjectable from "../../../../common/app-paths/directory-for-user-data/directory-for-user-data.injectable";
 import directoryForDownloadsInjectable from "../../../../common/app-paths/directory-for-downloads/directory-for-downloads.injectable";
-import getConfigurationFileModelInjectable
-  from "../../../../common/get-configuration-file-model/get-configuration-file-model.injectable";
-import appVersionInjectable
-  from "../../../../common/get-configuration-file-model/app-version/app-version.injectable";
+import getConfigurationFileModelInjectable from "../../../../common/get-configuration-file-model/get-configuration-file-model.injectable";
+import appVersionInjectable from "../../../../common/get-configuration-file-model/app-version/app-version.injectable";
+import fetchInjectable from "../../../../common/fetch/fetch.injectable";
+import fetchMock, { type FetchMockSandbox } from "fetch-mock";
+
+console.log("This is here as a reminder that mockFs breaks things and needs to be removed");
 
 mockWindow();
 
 jest.setTimeout(30000);
 jest.mock("fs-extra");
 jest.mock("../../notifications");
-
-jest.mock("../../../../common/utils/downloadFile", () => ({
-  downloadFile: jest.fn(({ url }) => ({
-    promise: Promise.resolve(),
-    url,
-    cancel: () => {},
-  })),
-  downloadJson: jest.fn(({ url }) => ({
-    promise: Promise.resolve({}),
-    url,
-    cancel: () => { },
-  })),
+jest.mock("../../../../common/utils/tar", () => ({
+  listTarEntries: () => [] as string[],
 }));
```

Comment:
New code should `di.override` instead of monkey-patch. I suspect there's a reason for this temptation. Let's take that discussion offline?

Reply:
Not wanting to change everything in this PR

Reply:
Removed this mock as it is not needed anymore